src/playerClass.ts

```diff
@@ -0,0 +1,22 @@
+import { Pokemon } from "./pokemonClass";
+
+export class Player {
+  private _name: string;
+  private _pokemons: Pokemon[];
+  constructor(name: string, pokemons: Pokemon[]) {
+    if (name.length > 8) {
+      throw new Error("Too long name");
+    }
+    if (pokemons.length != 3) {
+      throw new Error("Wrong number of pokemons");
+    }
+    this._name = name;
+    this._pokemons = pokemons;
+  }
+  get name(): string {
+    return this._name;
+  }
+  get pokemons(): Pokemon[] {
+    return this._pokemons;
+  }
+}
```

Comment:
I was thinking that we might need some method to getAlivePokemons() and it could be a good idea to place it in the Player class. It will simply use isAlive() method from pokemon class.